### PR TITLE
app-emulation: update qemu to 5.2.0

### PR DESCRIPTION
**Pull request by sayanchowdhury**
# app-emulation: update qemu to 5.2.0

# How to use

```bash
emerge-amd64-usr app-emulation/qemu
```

# Testing done
[Jenkins Build](http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/2404/cldsv/)

# Update

Another [Jenkins Build](http://jenkins.infra.kinvolk.io:8080/job/os/job/manifest/2494/cldsv) is running.

### CODE REVIEW

File: app-emulation/qemu/qemu-5.2.0-r2.ebuild [hunk 1]
```diff
@@ -1,13 +1,11 @@
-# Copyright 1999-2020 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI="7"
 
-PYTHON_COMPAT=( python{3_6,3_7} )
+PYTHON_COMPAT=( python3_7 )
```
Comment (dongsupark) [inline app-emulation/qemu/qemu-5.2.0-r2.ebuild:6]: Not sure about the change.

Without `python3_6`, the build fails like:
```
  The following REQUIRED_USE flag constraints are unsatisfied:
    python_targets_python3_7
```

**Review by dongsupark — COMMENTED** [review]
Shouldn't we probably disable `slirp` for `app-emulation/qemu` in `profiles/coreos/base/package.use`?

Otherwise it fails like:
```
emerge: there are no ebuilds to satisfy "net-libs/libslirp".
(dependency required by "app-emulation/qemu-5.2.0-r2::coreos[-static,slirp]" [ebuild])
(dependency required by "qemu" [argument])
```

File: app-emulation/qemu/qemu-5.2.0-r2.ebuild (hunk 1 above)
Comment (sayanchowdhury) [reply app-emulation/qemu/qemu-5.2.0-r2.ebuild:6]: I did fix both these issues when fixing the build issues. I'll have a look.

**Comment by pothos** [comment]
Should work, on Jenkins we don't use slirp because we run kola `qemu` tests and not `qemu-unpriv`.

**Comment by dongsupark** [comment]
Looks good, Thanks!

**Comment by sayanchowdhury** [comment]
The updated Jenkins CI build passed. Marked this ready for review again.

**Review by pothos — APPROVED** [review]

**Pull request merged**